modules/install-upgrade/pages/production-cluster-sizing.adoc

```diff
@@ -0,0 +1,216 @@
+:activeTopics: 1
+:messageSize: 1000
+:messageThroughput: 100000
+:retentionPolicy: 3600
+:ttlPolicy: 24
+:tieredStoragePolicy: n/a
+:messageReplicationFactor: 3
+
+= Production Cluster Sizing
+
+This document summarizes DataStax's recommendations for the sizing and optimization of Apache Pulsars cluster on Linux in a production environment.
+
+Of course, the sizing of a cluster depends on factors like use case and expected load, so this document is not intended to be a one-size-fits-all guide. Rather, we'd like to demonstrate how we consider and solve the problems inherent in sizing Pulsar clusters, and assist you on your journey to unlocking the scaling power of Pulsar.
+
+This page summarizes the requirements, assumptions, definitions, and methodologies that inform our cluster sizing recommendations.
+For deployment-specific recommendations, see the following pages:
+
+* xref:production-cluster-vm.adoc[]
+* xref:production-cluster-multiregion.adoc[]
+* xref:production-cluster-sandbox.adoc[]
+
+== Dedicated VMs or Kubernetes?
+
+As you begin your journey to design an Apache Pulsar cluster, one of the first questions to consider is what infrastructure your cluster will run on.
+Most of this guide will focus on running a cluster with dedicated virtual machines.
+While Kubernetes is the more popular option, it is easier to express disk calculations, throughput, and secure communications in terms of a VM.
+We will call out Kubernetes specifics as they arise in the guide.
+
+== Pulsar cluster components
+
+Pulsar clusters come in many shapes and sizes. There are minimum components for base functionality, and there are some recommended add-on components that make management and observability easier. For this guide we will focus on the required components and what it takes to make them resilient to outages and highly available in a three-zone cloud.
+
+=== Required components
+
+* Zookeeper - This is Pulsar’s meta data store. It stores data about a cluster’s configuration, helps the proxy direct messages to a broker, and holds Bookie configurations. To star, use 1 instance of Zookeeper in your cluster and let the number of lookups delegate scaling from there.
```

Comment:
Typo - 'To Star, use' should be 'To Start'

Reply:
You need at least 3 instances of Zookeeper for quorum. Quorum size of (n/2 + 1).  This configuration avoids a 'split-brain' scenario.  I'm not 100% sure if you can deploy a single ZK, if it's possible, it wouldn't be something we recommend.